alf/nest/utils.py

```diff
@@ -257,6 +257,27 @@ def _convert_cpu(tensor):
         raise NotImplementedError("Unknown device %s" % d)
 
 
+def add_batch_info(experience, batch_info, buffer=()):
```

Comment:
This is not for general nest. Perhaps it should be put into data_structures.py

Reply:
done